3-reveal/migrations/2-transaction_tables/deploy/index_event_form_times.psql

```diff
@@ -0,0 +1,11 @@
+-- Deploy reveal_transaction_tables:index_event_form_times to pg
+-- requires: events
+
+BEGIN;
+
+SET search_path TO :"schema",public;
+
+CREATE INDEX IF NOT EXISTS events_form_start_desc_idx ON events ((form_data->>'start') DESC);
```

Comment:
Does the DESC impact performance in any way?  If it doesn't it could be reasonable to simplify and drop, but could be wrong.